src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -515,6 +587,30 @@ public virtual RedirectToActionResult RedirectToAction(string actionName, string
         }
 
         /// <summary>
+        /// Redirects (<see cref="StatusCodes.Status307TemporaryRedirect"/>) to the specified action with 
+        /// <see cref="RedirectToActionResult.Permanent"/> set to false and <see cref="RedirectToActionResult.PreserveMethod"/> 
+        /// set to true, using the specified <paramref name="actionName"/>, <paramref name="controllerName"/>, 
+        /// <paramref name="routeValues"/>, and <paramref name="fragment"/>.
+        /// </summary>
+        /// <param name="actionName">The name of the action.</param>
+        /// <param name="controllerName">The name of the controller.</param>
+        /// <param name="routeValues">The parameters for a route.</param>
```

Comment:
It is inconsistent throughout MVC. A lot of instances have better phrasing like [this](https://github.com/aspnet/Mvc/blob/c2fee4d74d642ade9cf513fc60498aafcb51f9c0/src/Microsoft.AspNetCore.Mvc.Core/AcceptedAtActionResult.cs#L25).

But in [this](https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs#L635) class and some others, it is phrased this way.